cpu/xive.py

```diff
@@ -52,6 +54,14 @@ def setUp(self):
 
         self.no_config_parameter = []
 
+        self.detected_distro = distro.detect()
+        if self.detected_distro.name not in ['SuSE', 'rhel']:
+            self.cancel('Unsupported OS %s' % self.detected_distro.name)
+        if (self.detected_distro.name == 'SuSE'):
+            self.device = "eth0"
+        elif (self.detected_distro.name == 'rhel'):
+            self.device = "net0"
+
```

Comment:
This does not seem related to the changes being proposed for XIVE. Can you remove it ?

Reply:
> This does not seem related to the changes being proposed for XIVE. Can you remove it ?

Looks like I checked-in the code changes which were intended for a different scenario that I was working on, in the same file. Sorry, will get it removed.

Reply:
> > This does not seem related to the changes being proposed for XIVE. Can you remove it ?
> 
> Looks like I checked-in the code changes which were intended for a different scenario that I was working on, in the same file. Sorry, will get it removed.

Done.